### PR TITLE
Oci8 removes old PHP5 related description

### PR DESCRIPTION
**Pull request by mowangjuanzi**

### CODE REVIEW

File: reference/oci8/constants.xml [hunk 1]
```diff
@@ -41,8 +41,7 @@
          <entry><constant>OCI_CRED_EXT</constant></entry>
          <entry>
            Used with <function>oci_connect</function> for using
-           Oracles' External or OS authentication.  Introduced in PHP
-           5.3 and PECL OCI8 1.3.4.
+           Oracles' External or OS authentication.  Introduced in PECL OCI8 1.3.4.
```
Comment (Girgias) [inline reference/oci8/constants.xml:44]: I think at this point one can just remove the "Introduced in PECL" version as they are that old and nobody should use them (as they don't even work on PHP 7)

**Review by Girgias — CHANGES_REQUESTED** [review]
Please also remove mentions of the old PECL version, otherwise LGTM

File: reference/oci8/constants.xml (hunk 1 above)
Comment (mowangjuanzi) [reply reference/oci8/constants.xml:44]:
```
// from http://pecl.php.net/package/oci8
Use 'pecl install oci8-2.2.0' to install for PHP 7.
```

So I remove the PECL OCI8 < 2.2.0 description?

Comment (Girgias) [reply reference/oci8/constants.xml]: Yes please!

**Review by Girgias — APPROVED** [review]

**Comment by mowangjuanzi** [comment]
Is it possible to merge? 😁

**Comment by Girgias** [comment]
Sorry, thought you had merge rights on doc-en. .-.

**Comment by mowangjuanzi** [comment]
Haha, I only have doc-zh.

**Pull request merged**